extensions/amp-story/1.0/amp-story-interactive.js

```diff
@@ -168,6 +175,8 @@ export class AmpStoryInteractive extends AMP.BaseElement {
 
     /** @const @protected {!./variable-service.AmpStoryVariableService} */
     this.variableService_ = getVariableService(this.win);
+
+    this.updateStoryStoreState_(null);
```

Comment:
So, this logic I used to have it as a method from amp-story.js where it would `querySelector("amp-story-interactive-quiz, amp-story-interactive-poll, amp-story-interactive-binary-poll")`, and then `getImpl()` and then call the method. If we want to decouple amp-story from this interactives extension, I assumed that it was a good idea to not rely on outside code (amp-story.js) to initialize these components.

Check the other way of implementing this on the [previous branch I had](https://github.com/mszylkowski/amphtml/blob/956635004f6ef0eaf13e91b34297cf616b8ef911/extensions/amp-story/1.0/amp-story.js#L1071). If that's better, I'll use it instead of this.

Reply:
Actually I'm currently working on splitting the interactive components into their own extensions, I should initialize them properly after lazy-loading the extension. I can move this line into amp-story when I do that in a future PR when I initialize them properly

Reply:
Sounds good, let's figure this out in a follow up PR. Doing `getImpl()` is definitely not a good idea though. I have other solutions in mind that we can discuss when it's time.